README.md

```diff
@@ -98,6 +98,13 @@ details.
 **NOTE:** Generators run in RSpec 3.x will now require `rails_helper` instead
 of `spec_helper`.
 
+### `rails_helper` vs `spec_helper`
```

Comment:
Is this remains in this part of the documentation, it should really be included with the above as it's a kind of upgrade note.